videorender.cpp

```diff
@@ -1635,3 +2075,34 @@ void cVideoRender::SetVideoOutputPosition(const cRect &rect)
 
 	LOGDEBUG("videorender: %s: %d %d %d %d%s", __FUNCTION__, rect.X(), rect.Y(), rect.Width(), rect.Height(), m_videoIsScaled ? ", video is scaled" : "");
 }
+
+/**
+ * Enable displaying pip on screen
+ *
+ * @param on         true, if pip should be displayed
+ */
+void cVideoRender::SetEnablePip(bool on)
```

Comment:
Name the setter like the member for consistency?
```suggestion
void cVideoRender::SetPipActive(bool active)
```